src/components/TweetCreationForm.tsx

```diff
@@ -37,12 +43,22 @@ const StyledPopover = styled(Popover)`
 `;
 
 const TweetCreationForm: React.FC = () => {
+  const [content, setContent] = useState("");
+
+  const dispatch = useDispatch<AppDispatch>();
+  const { isLoading } = useSelector<
+    RootState,
+    RootState["tweets"]["feedTweets"]
+  >((store) => store.tweets.feedTweets);
```

Comment:
to subscribe only for one variable, you can do it directly:
```
const isLoading = useSelector<
    RootState,
    boolean
  >((store) => store.tweets.feedTweets.isLoading);
```

Reply:
ok, will change